packages/mural-picker/src/components/card-list/index.tsx

```diff
@@ -88,11 +88,17 @@ export default class MuralCardList extends React.Component<PropTypes> {
       m => !m.favorite && m.id === this.props.selectedMural?.id,
     );
 
+    // If there aren't any favorite murals, we don't need the "All murals" title there.
+    const title =
+      this.props.murals.filter(mural => mural.favorite).length === 0
+        ? ' '
```

Comment:
Is there a reason not to make this an empty string?

Reply:
That's a great question! I almost put a comment there explaining why. 

If the string is empty, it removes the title from the html and the spacing is too close (for my taste). When I put the space for the title there, it keeps the little bit of breathing room between the cards and the `+ New mural` button. I'll take a screenshot comparing the two, so you can see what I mean.

Reply:
![image](https://user-images.githubusercontent.com/80708763/223515729-b3a38a24-a5b4-4a56-ab67-fd38cb560c6d.png)

Ideally the spacing would be somewhere in between these two to look more consistent with the other inputs, but this was the simple way to use existing styles.